Cargo.toml

```
@@ -9,13 +9,16 @@ readme = "README.md"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
+cached = "0.20.0"
 cargo_metadata = "0.9"
 docopt = "1"
 env_logger = "0.8"
 lol_html = "0.2"
 log = "0.4"
 num_cpus = "1.8"
+once_cell = "1.5.1"
 rayon = "1.0"
+regex = "1"
```

We only look at numbers, so we don't need unicode support.
```suggestion
regex = { version = "1", default-features = false, features = ["std", "perf"] }
```
Note that this removes support for \d, you'll have to switch to `[0-9]` instead.